tests/sentry/api/endpoints/test_auth_index.py

```diff
@@ -130,39 +139,73 @@ def test_superuser_no_sso(self):
             SUPERUSER_ORG_ID=self.organization.id
         ):
             self.login_as(user)
-            response = self.client.put(self.path, data={"password": "admin"})
```

Comment:
what happens if the user sends a password along with the superuser category/reason?

Reply:
removing the password was a copy and paste mistake on my part, this actually changes the behaviour of this test case so i will fix